src/storage/slab/slab.h

```diff
@@ -62,13 +62,15 @@ typedef struct {
     ACTION( slab_evict,         METRIC_COUNTER, "# slabs evicted"          )\
     ACTION( slab_memory,        METRIC_GAUGE,   "memory allocated to slab" )\
     ACTION( slab_curr,          METRIC_GAUGE,   "# currently active slabs" )\
-    ACTION( item_keyval_byte,   METRIC_GAUGE,   "key + val in bytes"       )\
-    ACTION( item_val_byte,      METRIC_GAUGE,   "value only in bytes"      )\
     ACTION( item_curr,          METRIC_GAUGE,   "# current items"          )\
-    ACTION( item_req,           METRIC_COUNTER, "# items allocated"        )\
-    ACTION( item_req_ex,        METRIC_COUNTER, "# item alloc errors"      )\
-    ACTION( item_insert,        METRIC_COUNTER, "# items inserted"         )\
-    ACTION( item_remove,        METRIC_COUNTER, "# items removed"          )
+    ACTION( item_alloc,         METRIC_COUNTER, "# items allocated"        )\
+    ACTION( item_alloc_ex,      METRIC_COUNTER, "# item alloc errors"      )\
+    ACTION( item_dealloc,       METRIC_COUNTER, "# items de-allocated"     )\
+    ACTION( item_linked_curr,   METRIC_GAUGE,   "# current items, linked"  )\
+    ACTION( item_link,          METRIC_COUNTER, "# items inserted to HT"   )\
+    ACTION( item_unlink,        METRIC_COUNTER, "# items removed from HT"  )\
+    ACTION( item_keyval_byte,   METRIC_GAUGE,   "key+val in bytes, linked" )\
+    ACTION( item_val_byte,      METRIC_GAUGE,   "value only in bytes"      )
```

Comment:
Yea it does. I wonder if this is a metric that's gonna be useful under normal operations. I feel people may care about it only when they receive a OOM (due to failure in eviction), which should be rare.